doc/man7/flux-broker-attributes.rst

```diff
@@ -33,15 +33,23 @@ version
    The version of flux-core that was used to build this broker.
 
 rundir [Updates: C]
-   A temporary directory where UNIX domain sockets and the default
-   content.backing-path are located (see below).  By default, each broker
-   rank creates a unique rundir in $TMPDIR and removes it on exit.  If
-   rundir is set on the command line, beware exceeding the UNIX domain socket
-   path limit described in :linux:man7:`unix`, as low as 92 bytes on
-   some systems.  If rundir is set to a pre-existing directory, the
-   directory is not removed on exit; if the broker has to create the
-   directory, it is removed.  In most cases this attribute should not
-   be set by users.
+   A temporary directory where the broker's UNIX domain sockets are located.
+   In addition, if ``statedir`` is not set, this directory is used by the
+   content backing store (if applicable).  By default, each broker rank creates
+   a unique ``rundir`` in ``$TMPDIR`` and removes it on exit.  If ``rundir`` is
+   set on the command line, beware exceeding the UNIX domain socket path limit
+   described in :linux:man7:`unix`, as low as 92 bytes on some systems.  To
+   support the :man1:`flux-start` ``--test-size`` option where multiple brokers
+   share a ``rundir``, if ``rundir`` is set to a pre-existing directory, the
+   directory is not removed by the broker on exit.  In most cases this
+   attribute should not be set by users.
+
+statedir [Updates: C]
+   A directory in which persistent state such as the content backing store data
+   is stored by the Flux broker, to facilitate restarts.  If unset, this data
```

Comment:
first sentence seems a little run on.  Perhaps

A directory in which persistent state is stored by the Flux broker.  For example, content backing store data will be stored here to facilitate restarts.